tests/Unit/CssInlinerTest.php

```diff
@@ -1369,10 +1370,12 @@ public function formattedCssDeclarationDataProvider(): array
             'one declaration with linefeed in property value' => [
                 "text-shadow:\n1px 1px 3px #000,\n1px 1px 1px #000;",
                 "text-shadow: 1px 1px 3px #000,\n1px 1px 1px #000;",
+                'text-shadow: 1px 1px 3px #000,1px 1px 1px #000;',
             ],
             'one declaration with Windows line ending in property value' => [
                 "text-shadow:\r\n1px 1px 3px #000,\r\n1px 1px 1px #000;",
                 "text-shadow: 1px 1px 3px #000,\r\n1px 1px 1px #000;",
+                'text-shadow: 1px 1px 3px #000,1px 1px 1px #000;',
```

Comment:
Those two don't seem to fit in the "… with (linefeed|Windows line ending) in property value" lists as they have neither. Maybe move them somewhere else? Or are the line endings missing?

Reply:
The first element is the property declaration for the input CSS.  The remaining element(s) are (possibilities for) the expected style attribute value.  We don't need to require that the line endings within the property value after the comma are preserved, so can allow the style attribute to be rendered with or without them.  (Currently they are preserved, but with Sabberworm's CSS parser they would be removed.)